tests/unit/test_charm.py

```diff
@@ -2,14 +2,15 @@
 # See LICENSE file for licensing details.
 
 import unittest
+from typing import Any, Dict
 from unittest.mock import patch
 
 import yaml
 from ops.testing import Harness
 
 from charm import PROMETHEUS_CONFIG, PrometheusCharm
 
-MINIMAL_CONFIG = {}
+MINIMAL_CONFIG: Dict[str, Any] = {}
```

Comment:
This is misleading because the values of the mapping is not really "Any" but actually a well defined but complex nested structure that would be horrendous to read if fully spec'd. For this reason I think it should be removed. If necessary we may remove MINIMAL_CONFIG itself since it seems to be empty and is probably a carry over after various refactorings.

Reply:
Then make sure it's not used and then delete it 👍🏽

Reply:
Removed.